wgpu-types/src/lib.rs

```diff
@@ -961,6 +963,37 @@ impl Default for Limits {
 
 impl Limits {
     /// These default limits are guaranteed to be compatible with GLES-3.1, and D3D11
+    /// 
+    /// Those limits are as follows:
```

Comment:
I think it would be nice to list those in terms of deviating fromi the regular default - same for the actual implementation

Reply:
1. In terms of specifying based on deviation in the implementation, that's already done for downlevel_webgl2_defaults but not downlevel_defaults. 2. I personally much prefer seeing all the limits listed like I did as you don't have to scroll back and forth if needed; it's all right there. But if you really feel like it would be better to list them in deviations from the default, I can change it.

Reply:
Maybe bold the limits that have changed from the previous step down?

Reply:
That's something I thought about just an hour ago but totally forgot about. I really want to do that as soon as I can. How should I mark it? * at the beginning of the line sound good?